influxdb3_wal/src/lib.rs

```diff
@@ -809,6 +856,10 @@ impl WalContents {
     pub fn is_empty(&self) -> bool {
         self.ops.is_empty() && self.snapshot.is_none()
     }
+
+    pub fn is_default(&self) -> bool {
+        self.min_timestamp_ns == i64::MAX && self.max_timestamp_ns == i64::MIN
```

Comment:
default for Gen1 is 10 minutes, so it should be a check that max_ns - min_ns = 600000000000